### ISSUE TITLE
MSFT-58[Registration] Show Button

### ISSUE THREAD

**Issue by veshu** [veshu/testjiraimport#112]
**Steps to Reproduce**
1. Go to https://accounts.topcoder.com/member/registration
2. Inspect Element
3. Observe the

**Expected Result**
Expected contrast ratio of 4.5:1. 

**Actual Result**
Element has insufficient color contrast of 1.37 (foreground color: #FFFFFF, background color: ##DCDCE0, font size: small.

**Device**: Desktop/Labtop
**Operating System**: macOS
**Browser**: Chrome
**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.4.1 Use of Color

**Attachment**
